core/src/main/java/com/google/errorprone/bugpatterns/MutablePublicArray.java

```diff
@@ -67,7 +67,7 @@ private static boolean nonEmptyArrayMatcher(VariableTree arrayExpression, Visito
       return false;
     }
     JCNewArray newArray = (JCNewArray) arrayExpression.getInitializer();
-    if (!newArray.getDimensions().isEmpty()) {
```

Comment:
Thanks for the fix. I suggest just doing the null-check here, instead of having to repeat it below:

```
    if (newArray == null) {
      return false;
    }
```

(I can make that edit when importing the PR, this is just an FYI.)